src/analysis/retail/druid/balance/CHANGELOG.tsx

```diff
@@ -5,7 +5,8 @@ import { Hartra344, Sref, ToppleTheNun, ap2355, attluh, Jundarer } from 'CONTRIB
 import { SpellLink } from 'interface';
 
 export default [
-  change(date(2023, 3, 16), <>Update FotF, dot refreshes, rattle, sotf, filler usage and other small inaccuaries.</>, Jundarer),
+  change(date(2023, 8, 20), <>Update for 10.1.5 and show Eclipses on timeline.</>, Jundarer),
+  change(date(2023, 7, 16), <>Update FotF, dot refreshes, rattle, sotf, filler usage and other small inaccuaries.</>, Jundarer),
```

Comment:
```suggestion
  change(date(2023, 7, 16), <>Update FotF, dot refreshes, rattle, sotf, filler usage and other small inaccuracies.</>, Jundarer),
```
Since you're already adjusting this entry, might as well fix a spelling mistake